server.py

```diff
@@ -231,4 +254,9 @@ def get_data_from_neighbors():
 
 site = twserver.Site(Simple())
 reactor.listenTCP(int(config.get('port', 8080)), site)
+
```

Comment:
Is this still true - or did I see a reset at line 72

Reply:
that one worked, this one doesn't.  i want to reset to allow the file to be moved, but not yet sure how.  i'll do that in another branch at some point (#108 )